dev/bots/utils.dart

```diff
@@ -191,6 +202,9 @@ Never reportErrorsAndExit(String message) {
   }
   print(redLine);
   print('You may find the errors by searching for "╡ERROR #" in the logs.');
+  if (dryRun) {
```

Comment:
Reading this code I am not sure why dryrun would want a different error here.

Reply:
This is unused code, removed.